TelCo.ColorCoder/ColorPairService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// Service to mapping of pair number to color and color to pair number.
+    /// </summary>
+    class ColorPairService
+    {
+        
+        /// <summary>
+        /// Given a pair number function returns the major and minor colors in that order
+        /// </summary>
+        /// <param name="pairNumber">Pair number of the color to be fetched</param>
```

Comment:
there is a restriction on the pair number. It is useful to mention that here. the existing comment is implicitly understood from the method signature.